fem/pfespace.cpp

```diff
@@ -3555,6 +4968,37 @@ void ParFiniteElementSpace::Update(bool want_transform)
    }
 }
 
+void ParFiniteElementSpace::PRefineAndUpdate(const Array<pRefinement> & refs,
+                                             bool want_transfer)
+{
+   MFEM_VERIFY(PRefinementSupported(),
+               "p-refinement is not supported in this space");
+
+   if (want_transfer)
+   {
+      pfesPrev.reset(new ParFiniteElementSpace(pmesh, fec));
```

Comment:
This should account for `vdim` and `ordering`.

Reply:
Fixed in 7fe4c3c19943a1b5064aa33e99b357f41dbb75c7